src/containers/Header/index.js

```diff
@@ -39,6 +41,7 @@ export function Header(props) {
 		<header className={`application-header ${headerProperties ? 'header-props-open' : ""}`}>
 			{(props.app.props.hasSidebar || typeof props.app.props.hasSidebar == 'undefined') ?
 				<>
+
```

Comment:
@queuing4oranges I think this empty line is not necessary here, remove it please